src/ol_infrastructure/applications/bootcamps/__main__.py

```diff
@@ -153,10 +159,25 @@
 bootcamps_secrets = vault.Mount(
     "bootcamps-vault-secrets-storage",
     path="secret-bootcamps",
-    type="generic",
+    type="kv",
+    options={"version": 2},
     description="Static secrets storage for the bootcamps application",
+    opts=ResourceOptions(delete_before_replace=True),
+)
+
+bootcamps_vault_secrets = read_yaml_secrets(
+    Path(f"bootcamps/secrets.{stack_info.env_suffix}.yaml")
 )
 
+for key, data in bootcamps_vault_secrets.items():
+    secret_path = partial("{1}/{0}".format, key)
+    vault.kv.SecretV2(
+        f"bootcamps-vault-secrets-{key}",
+        name=key,
+        mount=bootcamps_secrets,
```

Comment:
Missing the `path` argument

Reply:
I tried it on vault-qa and for this `path` argument isn't needed - https://www.pulumi.com/registry/packages/vault/api-docs/kv/secretv2/